### PR TITLE
Governance

### PR DESCRIPTION
**Pull request by avanderbeek**
I have a few basic endpoints set up, but will tweak those as I work on the Hubble side of things.

### CODE REVIEW

File: lib/tasks/sync_governance.rake [hunk 1]
```diff
@@ -2,7 +2,7 @@ require "task_lock"
 
 task sync_governance: :environment do
   $stdout.sync = true
-  TaskLock.with_lock!(:sync) do
+  TaskLock.with_lock!(:sync_governance) do
```
Comment (rfunduk) [inline lib/tasks/sync_governance.rake:5]: can't do this without also updating `LOCKFILES`
https://github.com/figment-networks/bakerhub-indexer/blob/master/lib/task_lock.rb#L8

File: app/models/tezos/proposal.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    elsif elapsed_periods == 2
+        return self.passed_eval_period ? "testing period" : "rejected"
+    elsif elapsed_periods == 1
+        return self.passed_prop_period ? "evaluation voting period" : "rejected"
+    else
+        return "proposal period"
+    end
+  end
+
+end
```
Comment (rfunduk) [inline app/models/tezos/proposal.rb:27]: Indenting issues in this file

File: lib/tasks/sync_governance.rake [hunk 3]
```diff
@@ -0,0 +1,26 @@
+require "task_lock"
+
+task sync_governance: :environment do
+  $stdout.sync = true
+  TaskLock.with_lock!(:sync_governance) do
+    puts "Setting Tezos Chain"
+    chain = Tezos::Chain.first
```
Comment (rfunduk) [inline lib/tasks/sync_governance.rake:7]: Should it not go through all Tezos::Chain's? If there's always only 1 (which would be fine) then probably don't need the table at all eh?

Comment (avanderbeek) [reply lib/tasks/sync_governance.rake]: fixed to handle multiple chains.

**Pull request merged**